rules/rbe_repo.bzl

```diff
@@ -573,6 +579,11 @@ def _rbe_autoconfig_impl(ctx):
 
             # Create the versions.bzl file
             if ctx.attr.create_versions:
+                if not digest:
```

Comment:
I think it might be better to fail this earlier and have a more specific message on the problem. See my comments above.